bagua/torch_api/contrib/utils/__init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ["redis_store", "store"]
```

Comment:
I feel like all `utils` here should be excluded from sphinx API doc.

Reply:
> I feel like all `utils` here should be excluded from sphinx API doc.

I'd like to let users know the interface of `Store`.